app/src/main/graphql/com/hedvig/android/owldroid/graphql/offer.graphql

```diff
@@ -43,7 +44,38 @@ query Offer {
         }
       }
       insuranceCost {
-        ... CostFragment
+        ...CostFragment
+      }
+      perils(locale: $locale) {
+        ... PerilFragment
+      }
+      termsAndConditions(locale: $locale) {
+        displayName
+        url
+      }
+      insurableLimits(locale: $locale) {
+        label
+        limit
+        description
+      }
+      #      insuranceTerms(locale: $locale) {
```

Comment:
👀

Reply:
this and the termsBinder is commented out because it got this error `Error{message='Expected Iterable, but did not find one for field CompleteQuote.insuranceTerms.', locations=[Location{line=1, column=846}], customAttributes={path=[lastQuoteOfMember, insuranceTerms], extensions={code=INTERNAL_SERVER_ERROR, exception={message=Expected Iterable, but did not find one for field CompleteQuote.insuranceTerms.}}}}]
` that ill need to resolve

Reply:
We should make sure it's fixed upstream before we merge I think 🙂